gpoa/frontend/chromium_applier.py

```diff
@@ -101,7 +101,11 @@ def set_user_policy(self, name, obj):
                 logging.info(slogm('Set user ({}) property \'{}\' to {}'.format(self.username, name, obj)))
 
     def get_home_page(self, hkcu=False):
-        return self.get_hklm_string_entry('HomepageLocation')
+        response = self.get_hklm_string_entry('HomepageLocation')
+        result = 'about:blank'
```

Comment:
Just a nitpick (feel free to ignore):
```python
return self.get_hklm_string_entry('HomepageLocation') or 'about:blank'
```

Even better approach is to improve `get_hklm_string_entry` so it takes the 2nd argument, 
and returns it if the value is not set, similarly like `get`:

```python
d = {0: 'a', 1: 'b'}
x = d.get(0, 'foo') # a
y = d.get(2, 'foo') # foo
```

Reply:
Theres is `get_hklm_string_entry_default` function exists, but, yes, maybe it's better to rewrite these two functions into one function with parameter.